utils/stringUtils.ts

```diff
@@ -50,3 +50,9 @@ export function isValidUrl(url: string | URL) {
     }
     return url.protocol === 'https:'
 }
+
+// converts the theme name to sentence case
+export function toSentenceCase(theme: string) {
```

Comment:
We can remove this comment if we make the function name more explicit.

```suggestion
export function convertThemeNameToSentenceCase(theme: string) {
```